components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIProviderHostObject.java

```diff
@@ -426,6 +434,104 @@ public static NativeArray jsFunction_getBusinessOwners(Context cx, Scriptable th
         return myn;
     }
 
+    /**
+     * Retrieve business owner.
+     * @param cx      Rhino context
+     * @param thisObj Scriptable object
+     * @param args    Passing arguments
+     * @param funObj  Function object
+     * @return shared policy partials
+     * @throws org.wso2.carbon.appmgt.api.AppManagementException
+     */
+    public static NativeObject jsFunction_getBusinessOwner(Context cx, Scriptable thisObj, Object[] args,
+                                                           Function funObj) throws AppManagementException {
+        if (args == null || args.length != 1) {
+            handleException("Invalid number of input parameters.");
+        }
+        if (args[0] == null || args[0] == "null") {
+            handleException("Error while reading business owner. Owner id is null");
+        }
+        NativeObject row = new NativeObject();
+        int ownerId;
+        try {
+            ownerId = Integer.valueOf(args[0].toString());
+            APIProvider apiProvider = getAPIProvider(thisObj);
+            BusinessOwner businessOwner = apiProvider.getBusinessOwner(ownerId);
+            row.put("businessOwnerId", row, businessOwner.getBusinessOwnerId());
+            row.put("businessOwnerId", row, businessOwner.getBusinessOwnerId());
+            row.put("businessOwnerName", row, businessOwner.getBusinessOwnerName());
+            row.put("businessOwnerEmail", row, businessOwner.getBusinessOwnerEmail());
+            row.put("businessOwnerDescription", row, businessOwner.getBusinessOwnerDescription());
+            row.put("businessOwnerSite", row, businessOwner.getBusinessOwnerSite());
+            List<BusinessOwnerProperty> businessOwnerPropertiesList = businessOwner.getBusinessOwnerPropertiesList();
+            if(businessOwnerPropertiesList != null) {
+                JSONObject businessOwnerPropertiesObject = new JSONObject();
+                for (int i = 0; i < businessOwnerPropertiesList.size(); i++) {
+                    JSONObject businessOwnerPropertyObject = new JSONObject();
+                    BusinessOwnerProperty businessOwnerProperty = businessOwnerPropertiesList.get(i);
+                    businessOwnerPropertyObject.put("propertyValue", businessOwnerProperty.getPropertyValue());
+                    businessOwnerPropertyObject.put("isShowingInStore", businessOwnerProperty.isShowingInStore());
+                    businessOwnerPropertiesObject.put(businessOwnerProperty.getPropertyId(),businessOwnerPropertyObject);
+                }
+                row.put("businessOwnerProperties", row, businessOwnerPropertiesObject.toJSONString());
+            } else {
+                row.put("businessOwnerProperties", row, null);
+            }
+        } catch (NumberFormatException e) {
+            log.warn("Business owner id : " + args[0] + " is not an integer.", e);
+        }
+        return row;
+    }
+
+    /**
+     * Search business owners with pagination
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files